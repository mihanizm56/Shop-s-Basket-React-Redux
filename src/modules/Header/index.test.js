import React from "react"
import { shallow } from "enzyme"
import { Header } from "./index"

describe("Header test", () => {
  it("Renders correct", () => {
    const component = shallow(<Header />)

    expect(component).toMatchSnapshot()
  })
})
