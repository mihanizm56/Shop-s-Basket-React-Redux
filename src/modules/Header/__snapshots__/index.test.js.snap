// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header test Renders correct 1`] = `
<styled.div>
  <styled.div>
    <styled.div>
      <Number />
      <FreeCall />
    </styled.div>
    <styled.div>
      <styled.div>
        <styled.div />
        <Star />
      </styled.div>
      <styled.div>
        <styled.div />
        <Youtube />
      </styled.div>
      <styled.div>
        <styled.div />
        <LiveStream />
      </styled.div>
    </styled.div>
  </styled.div>
</styled.div>
`;
